### PR TITLE
Add/stats to jetpack complete lightbox

### PR DESCRIPTION
**Pull request by IanRamosC**
## Proposed Changes

* Update Jetpack Complete lightbox to include Jetpack Stats
* Update the comparison page to have Stats as a product, not just a feature

image
image

## Testing Instructions

* Do one of the following:
  * Open the Jetpack Cloud live link
  * `git checkout add/stats-to-jetpack-complete-lightbox` and `yarn start-jetpack-cloud` locally
* Visit `/pricing#jetpack_complete` and confirm that you see a full item for Stats in the "Products included" list 
* Confirm that Stats is also in the sidebar list, with the correct price ($10 if your currency is USD)
* Visit `/features/comparison`, confirm that Stats under the "Growth" section links to https://jetpack.com/stats, and that it matches the screenshot above

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: packages/calypso-products/src/plans-list.tsx [hunk 1]
```diff
@@ -3050,6 +3051,7 @@ export const PLANS_LIST: Record< string, Plan | JetpackPlan | WPComPlan > = {
 			PRODUCT_JETPACK_BOOST,
 			PRODUCT_JETPACK_SOCIAL_ADVANCED,
 			PRODUCT_JETPACK_SEARCH,
+			PRODUCT_JETPACK_STATS_MONTHLY,
```
Comment (IanRamosC) [inline packages/calypso-products/src/plans-list.tsx]: This will be later changed to Stats yearly

Comment (CodeyGuyDylan) [reply packages/calypso-products/src/plans-list.tsx]: I just approved https://github.com/Automattic/wp-calypso/pull/80320 which adds `PRODUCT_JETPACK_STATS_YEARLY`, so you may be able to get that in here before deploying if that one gets deployed

**Review by CodeyGuyDylan — APPROVED** [review]
Looks good, tested well, left one comment about the yearly constant, but it's nonblocking

File: packages/calypso-products/src/plans-list.tsx (hunk 1 above)
Comment (CodeyGuyDylan) [reply packages/calypso-products/src/plans-list.tsx]: Looks like it just got merged 😄

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8701621

Thank you @IanRamosC for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8701621).

**Pull request merged**